sourcemash/api/users.py

```diff
@@ -50,12 +49,14 @@ def __init__(self):
     def get(self):
         return { 'user': marshal(current_user, user_fields) }
```

Comment:
Email: null, id: 0 (both of which don’t exist and pass our HTML/JS checks as expected)